darts/timeseries.py

```diff
@@ -308,7 +309,25 @@ def from_dataframe(df: pd.DataFrame,
 
         # get time index
         if time_col:
```

Comment:
Btw, if we rule out `if time_col is None:` and `if time_col not in df.columns:` we could reduce the indentation level quite a bit and make it easier to read:

```suggestion
        if time_col is None:
            raise_if_not(isinstance(df.index, pd.Int64Index) or isinstance(df.index, pd.DatetimeIndex),
                         'If time_col is not specified, the DataFrame must be indexed either with'
                         'a DatetimeIndex, or with a Int64Index (incl. RangeIndex).', logger)
    
        if time_col not in df.columns:
            raise_log(AttributeError('time_col=\'{}\' is not present.'.format(time_col)))
    
        if np.issubdtype(df[time_col].dtype, object):
            ...
```

Reply:
Note the different error strings. Reduce level → reduce verbosity.

Reply:
Could you please explain a bit more what that means (out of curiosity)? That the error message in the order from before are more clear/verbose?